auth/template.yaml

```diff
@@ -1948,6 +1948,7 @@ Resources:
             - !FindInMap [SharedSignalsJwksUri, Environment, !Ref Environment]
             - !Sub https://${MockSharedSignalsTransmitterApi}.execute-api.${AWS::Region}.amazonaws.com/${Environment}/.well-known/jwks.json
           JWKS_CACHE_DURATION: 600000 # ten minutes in ms
+          ENABLE_SHARED_SIGNAL: !Sub '{{resolve:ssm:/${ConfigStackName}/feature-flags/shared-signal}}'
```

Comment:
Can we add an acceptance test to demonstrate this works in a deployed environment? 

For instance, you can invoke a lambda with different environment variables - can we create a test that invokes this lambda with a disabled feature flag and assert the response is correct?